### ISSUE TITLE
Issues using mesa3d

### ISSUE THREAD

**Issue by Torje3000** [pal1000/mesa-dist-win#198]
Sorry for using the issues page for a personal problem, but I find no other contact info or documentation that I can understand.

How do I use Mesa3D?
Say if I want to run a OpenGL 4.3 program (blender 4.1.0) on a GPU (and drivers) that only support up 2.1, am I wrong expecting it to work with Mesa3D? 

@set MESA_GL_VERSION_OVERRIDE=4.3COMPAT
@set MESA_GLSL_VERSION_OVERRIDE=430
@blender.exe

I run a Intel Core i5 540M, with its integrated gpu

**Comment by pal1000** [comment]
> Say if I want to run a OpenGL 4.3 program (blender 4.1.0) on a GPU (and drivers) that only support up 2.1, am I wrong expecting it to work with Mesa3D?

No, it should work.. Blender 4.1.0 x64 worked for me with Mesa3D 24.2.2 MSVC release.

> @set MESA_GL_VERSION_OVERRIDE=4.3COMPAT
@set MESA_GLSL_VERSION_OVERRIDE=430

With current Mesa3D releases you don't need to override OpenGL support unless you need bleeding edge  OpenGL 4.6, so blender doesn't need this.

**Comment by Torje3000** [comment]
Tried using per-app deployment tool, with 24.2.2 MSVC Release. Blender just closes immediately after launch now, no errors or anything

**Comment by StefanG3D** [comment]
Launch Blender in debug mode to hopefully get some hints:
`blender_debug_gpu.cmd`

**Comment by Torje3000** [comment]
I got an empty log file
